scripts/pylib/twister/twisterlib.py

```diff
@@ -2083,6 +2090,7 @@ def __init__(self, testcase, platform, outdir):
         self.metrics = dict()
         self.handler = None
         self.outdir = outdir
+        self.run_id = random.getrandbits(64)
```

Comment:
counter is a very bad idea, just consider how this would work if you are running just one test, you always get the same number, i.e run `tests/arch/arm/arm_runtime_nmi/` on a device multiple times.....
getrandbits was used for proofing only, we need a mix of randomness and something that has data from the test itself, so for example, md5(instance name) + random bits....